modules/ROOT/pages/mule-digital-signature-processor.adoc

```diff
@@ -646,9 +646,24 @@ The following procedure describes how to verify digital signatures on messages a
 <signature:verify-signature config-ref="" doc:name="Signature"/>
 ----
 
+[start=3]
 . Use the *Using* field (or `using` attribute in XML) to indicate the type of signature:` JCE_SIGNER` or `XML_SIGNER`.
 . Optionally, enter a Mule expression in the *Input Reference* field to indicate the part of the message payload to which the signature applies. In other words, a signature may apply to only part of the message payload.
-. In the *Expected Signature* field, enter a Mule expression that Mule can use to compare and verify that the signature on a message it received is authentic.
+. In the *Expected Signature* field, enter a Mule expression that Mule can use to compare and verify that the signature on a message it received is authentic. +
+. Optionally, select *Fail On Verification* option if an exception must be thrown when the signature mismatch the expected one. Defaults to false.
```

Comment:
"...select the *Fail On Verification* option..."
"mismatch" > "mismatches" even better "does not match"